src/components/with-public-services/redux/reducer.ts

```diff
@@ -74,6 +84,42 @@ export default function reducer(
         .set('publicServicesAggregations', null)
         .set('publicServicesPage', null)
         .set('publicServicesEvents', null);
+    case GET_PUBLIC_SERVICES_REQUIRED_BY_REQUESTED:
+      return state;
```

Comment:
Reset public services
```suggestion
      return state.set('publicServicesRequiredBy', fromJS([]));
```